### ISSUE TITLE
Delay validation if home assistant is not running/started yet?

### ISSUE THREAD

**Issue by xconverge** [zachowj/node-red-contrib-home-assistant-websocket#1510]
### Describe the bug

With the validation changes, when my server starts up, I think node-red beats home assistant in a race to startup, causing there to be many many validation errors in this plugin. Should there be a conditional check somewhere on if the socket is open before marking things as invalid?

I am not really sure if it matters or what to do since technically they are invalid nodes at the time..

For now I will just restart node-red once everything is up and going, and all of the validation goes away

### Environment Information

0.68.1

**Comment by zachowj** [comment]
It does wait until Home Assistant has sent over at least the first batch of data. I could also check that Home Assistant is in its "running state"

https://github.com/zachowj/node-red-contrib-home-assistant-websocket/blob/af9e7289f58ee2ee8c39ba4f51fe83e6cf1c19d4/src/homeAssistant/Websocket.ts#L299-L314

**Comment by zachowj** [comment]
What types of issues are being reported that are not issues? entity IDs?

**Comment by xconverge** [comment]
Interesting, I can't synthetically cause this and it works fine

starting them out of order
```
docker compose stop nodered
docker compose stop homeassistant
docker compose start nodered
docker compose start homeassistant
```

starting both at same time
```
docker compose stop nodered
docker compose stop homeassistant
docker compose up -d
```

If I stop the entire stack (~20 containers) and start it (more cpu intensive?)

```
docker compose stop
docker compose up -d
```

You can see the connection is up from the nodes on the left but the validation still stays invalid (on the right)

![image](https://github.com/user-attachments/assets/68671ec7-4ad1-42fd-a0a3-4d8b566b1d8f)

The periodic checking never changes the invalid stuff on the right

If I move an impacted node and redeploy, it also does not go away

Restarting node red fixes this immediately

**Comment by xconverge** [comment]
all 25 are only `sun.sun` or `notify.mobile_app_pixel_8` which is strange

**Comment by xconverge** [comment]
And a few minutes later the sun.sun ones went away so I see you are right in that as data changes they go away!

Now I am left with 18 that are all notify.mobile_app_pixel_8, I will see if they go away

**Comment by xconverge** [comment]
I didn't see that the periodic was 6 hours, I just assumed (for no reason...) it was minutes. I will leave the system as is, but I am guessing it will resolve itself in 6 hours and that there isn't really much worth doing here!

**Comment by zachowj** [comment]
When you can, try v0.68.3 to see if that eliminates the issue.

**Comment by xconverge** [comment]
> When you can, try v0.68.3 to see if that eliminates the issue.

Looks like it eliminates the issue! All good now!

**Comment by xconverge** [comment]
Fixed by fdda4fd7916dcf44a45d61acd91e577e72922d0b
